bundles/best_practices.rst

```diff
@@ -404,10 +404,10 @@ The ``composer.json`` file should include at least the following metadata:
 
 ``license``
     ``MIT`` is the preferred license for Symfony bundles, but you can use any
-    other license.
+    license.
 
 ``autoload``
-    This information is used by Symfony to load the classes of the bundle. The
+    This information is used by Symfony to load the classes in the bundle. The
```

Comment:
This change looks wrong to me. Autoloading is about how classes provided by the bundle are loaded. So we are actually talking about "classes **of** the bundle".